docs/how-to/deploy_cups_sdcore.md

```diff
@@ -0,0 +1,217 @@
+
+| Name | IP Address | Purpose |
+| ---- | ---------- | ------- |
+| juju-controller.k8s.priv | 10.201.0.10 | Management address for Juju machine
+| control-plane.k8s.priv | 10.201.0.11 | Management address for control plane cluster machine | 
+| user-plane.k8s.priv | 10.201.0.12 | Management address for user plane cluster machine | 
+| control-plane.juju.priv | 10.201.0.101 | Juju controller load balancer port |
+| upf.core.priv | 10.201.0.151 | IP address for UPF to bind to |
+| amf.core.priv | 10.201.0.201 | Externally reachable control plane endpoint for the AMF |
+
+
+`````{tab-set}
+
+````{tab-item} Bootstrap
+
+This section covers how to install Juju and Microk8s to act as the infrastructure for SD-Core, and may be skipped as needed.  If existing Kubernetes clouds are going to be used, Juju still must be bootsrapped using the appropriate `kubeconfig.yaml` file
+
+----------------------
+#### SD-Core Control Plane
+
+All commands in this section are run on the SD-Core Control Plane Kubernetes cluster host.
+
+Install Microk8s and configure Metallb to expose 1 IP address for the AMF:
+
+```bash
+sudo snap install microk8s --channel=1.27-strict/stable
+sudo microk8s enable hostpath-storage
+sudo microk8s enable metallb:10.201.0.201-10.201.0.201
+```
+
+Export the Kubernetes configuration and copy that to the controller:
+
+```bash
+sudo microk8s.config > control-plane-cluster.yaml
+scp control-plane-cluster.yaml juju-controller.k8s.priv:
+```
+
+----------------------
+#### SD-Core User Plane
+
+All commands in this section are run on the SD-Core Control Plane Kubernetes cluster host.
+
+Install Microk8s, configure Metallb to expose 1 IP address for the UPF, and add the Multus plugin:
+
+```bash
+sudo snap install microk8s --channel=1.27-strict/stable
+sudo microk8s enable hostpath-storage
+sudo microk8s enable metallb:10.201.0.151-10.201-0.151
+sudo microk8s addons repo add community \
+    https://github.com/canonical/microk8s-community-addons \
+    --reference feat/strict-fix-multus
+sudo microk8s enable multus
+```
+
+Export the Kubernetes configuration and copy that to the controller:
+
+```bash
+sudo microk8s.config > user-plane-cluster.yaml
+scp user-plane-cluster.yaml juju-controller.k8s.priv:
+```
+
+If not done already, create macvlan interfaces for Kubernetes to use when accessing the `access` and `core` networks. In this how-to, the following network interfaces are available on the SD-Core User Plane Microk8s machine:
+
+- `ens3`: internal Kubernetes management interface.  This maps to the `management` subnet and must have an IP address.  In this how-to, the IP address should be `10.201.0.12`
+- `ens4`: access interface.  This maps to the `access` subnet, and does not have an IP on the host.
+- `ens5`: core interface.  This maps to the `core` subnet, and does not have an IP on the host.  Note that internet egress is required here and routing tables must be set to route gNB generated traffic 
```

Comment:
What do you think about adding an example routing table for that?

Reply:
I think it makes sense, but what shape could that take?  Just a table, or `iptables` commands?

Reply:
Removed from the howto as it is not relevant